contracts/PublicResolver.sol

```diff
@@ -52,4 +53,14 @@ contract PublicResolver is ABIResolver, AddrResolver, ContentHashResolver, DNSRe
         address owner = ens.owner(node);
         return owner == msg.sender || authorisations[node][owner][msg.sender];
     }
+
+    function multicall(bytes[] calldata datas) external returns(bytes[] memory rets) {
+        rets = new bytes[](datas.length);
```

Comment:
Please use more descriptive variable names. I don't know what `rets` stands for. Did you get inspiration from MakerDAO?